File: app/src/main/java/com/doordash/binding/viewmodel/MenuModel.java
package com.doordash.binding.viewmodel;

import androidx.databinding.BaseObservable;
import androidx.databinding.Bindable;
import androidx.databinding.ObservableArrayList;


import java.util.List;

public class MenuModel extends BaseObservable {

    @Bindable
    public ObservableArrayList<DishModel> dishes=new ObservableArrayList<>();

    public MenuModel(){
    }

    public void addAll(List<DishModel> list){
        dishes.addAll(list);
    }


}

File: app/src/main/java/com/doordash/binding/viewmodel/MenuModel.kt
package com.doordash.binding.viewmodel

import androidx.databinding.BaseObservable
import androidx.databinding.Bindable
import androidx.databinding.ObservableArrayList
import androidx.lifecycle.ViewModel
import com.doordash.bean.PopularItem

class MenuModel : ViewModel() {
    var dishes = ObservableArrayList<PopularItem>()//Bindable should move to get, works with   notifyPropertyChanged()
    fun addAll(list: List<PopularItem>) {
        dishes.addAll(list!!)
    }
}